surface/include/pcl/surface/gp3.h

```diff
@@ -288,28 +274,28 @@ namespace pcl
 
     protected:
       /** \brief The nearest neighbor distance multiplier to obtain the final search radius. */
-      double mu_;
+      double mu_{0.0};
 
       /** \brief The nearest neighbors search radius for each point and the maximum edge length. */
-      double search_radius_;
+      double search_radius_{0.0};
 
       /** \brief The maximum number of nearest neighbors accepted by searching. */
-      int nnn_;
+      int nnn_{100};
 
       /** \brief The preferred minimum angle for the triangles. */
-      double minimum_angle_;
+      double minimum_angle_{0.0};
```

Comment:
```suggestion
      double minimum_angle_{M_PI/18};
```

As per constructor - applies to multiple parameters.